json-logic/example.json

```diff
@@ -0,0 +1,318 @@
+{
+  "meta": {
+    "author": "Alexander",
+    "languages": [
+      "DE"
+    ],
+    "name": "Example",
+    "version": "0.1"
```

Comment:
Meta should include: id, date, version

Reply:
Should we also include "medical guidelines" on which the questionnaire is based?

Reply:
If we allow a questionnaire to build on an existing one, it would be good if the basic version is referenced in the metadata.

Reply:
> If we allow a questionnaire to build on an existing one, it would be good if the basic version is referenced in the metadata.

`derivedFrom` is used by FHIR

Reply:
> Meta should include: id, date, version

I don't think so, the id is no meta information. We could also shorten the discussion and just do it like FHIR (no `meta`, everything in the root object)

Reply:
Good point, I'll move `id` to top level.

@michael: I'm open to include additional information, please coordinate with @alexanderthieme which info is needed. 

I would not include referral to other questionnaires yet.

Reply:
Should also include API Version (Version of the file format).